### PR TITLE
SDDL43: Add bus markers

### PR DESCRIPTION
**Pull request by vangoespow**
Add bus markers into mobile view. This story is part 1 as it requires a custom renderer class to show a different icon.

### CODE REVIEW

**Review by shi24 — COMMENTED** [review]
Looks good to merge to me

File: DragonLoop/DragonLoopApp/DragonLoopApp/ViewModels/MapViewModel.cs [hunk 1]
```diff
@@ -35,13 +38,54 @@ public MapViewModel() : base(Settings.UrlBase)
...
+        public async Task ExecuteLoadBuses()
+        {
+            try
+            {
+                BusCollection.Clear();
+                await LoadBuses();
+                foreach (var bus in Buses)
+                {
+                    BusCollection.Add(bus);
+                    var pin = new Pin
```
Comment (ryancanavan) [inline DragonLoop/DragonLoopApp/DragonLoopApp/ViewModels/MapViewModel.cs]: I feel like we should tie adding the Bus pins to the route sliders just like you did for the stops right? It seems weird to show all the buses all the time and could get hard to see what the user actually cares about.

File: DragonLoop/DragonLoopApp/DragonLoopApp/ViewModels/MapViewModel.cs [hunk 2]
```diff
@@ -35,13 +38,54 @@ public MapViewModel() : base(Settings.UrlBase)
...
+        private async Task ExecuteLoadDataCommand()
         {
             if (IsBusy)
                 return;
 
             IsBusy = true;
 
+            try
+            {
+                await ExecuteLoadBuses();
```
Comment (ryancanavan) [inline DragonLoop/DragonLoopApp/DragonLoopApp/ViewModels/MapViewModel.cs]: I think you can use Task.WhenAll() here to run both loads at once. Currently it waits to load buses before it starts trying to load routes.

**Review by ryancanavan — CHANGES_REQUESTED** [review]
Pretty minor changes.

File: DragonLoop/DragonLoopApp/DragonLoopApp/ViewModels/MapViewModel.cs (hunk 1 above)
Comment (vangoespow) [reply DragonLoop/DragonLoopApp/DragonLoopApp/ViewModels/MapViewModel.cs]: ok i can do that

Comment (vangoespow) [reply DragonLoop/DragonLoopApp/DragonLoopApp/ViewModels/MapViewModel.cs]: This would require rewrite of all the code i have but i will be doing this

**Review by ryancanavan — APPROVED** [review]

**Pull request merged**